examples/event_handling/zoom_window.py

```diff
@@ -6,27 +6,27 @@
 This example shows how to connect events in one window, for example, a mouse
 press, to another figure window.
 
-If you click on a point in the first window, the z and y limits of the
-second will be adjusted so that the center of the zoom in the second
-window will be the x,y coordinates of the clicked point.
+If you click on a point in the first window, the z and y limits of the second
+will be adjusted so that the center of the zoom in the second window will be
+the x,y coordinates of the clicked point.
 
-Note the diameter of the circles in the scatter are defined in
-points**2, so their size is independent of the zoom
+Note the diameter of the circles in the scatter are defined in points**2, so
+their size is independent of the zoom.
 """
-from matplotlib.pyplot import figure, show
+
+from matplotlib import pyplot as plt
```

Comment:
`import matplotlib.pyplot as plt` (also in various other locations, please search/replace)

Reply:
fixed